### PR TITLE
Add Huawei P30 to devices with notch list

### PR DESCRIPTION
**Pull request by Taym95**
## Description

Fixed issue #709

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |     ❌      |
| Android |     ✅      |
| Windows |     ❌      |

## Checklist

* [x] I have tested this on a device/simulator for each compatible OS
* [ ] I added the documentation in `README.md`
* [ ] I mentioned this change in `CHANGELOG.md`
* [ ] I updated the typings files (`deviceinfo.d.ts`, `deviceinfo.js.flow`)
* [ ] I updated the dummy web/test polyfill (`default/index.js`)
* [ ] I added a sample use of the API (`example/App.js`)

### CODE REVIEW

File: deviceinfo.js [hunk 1]
```diff
@@ -87,6 +87,14 @@ const devicesWithNotch = [
...
     model: 'Mate 20 Pro',
   },
+  {
+    brand: 'Huawei',
+    model: 'P30 Lite',
+  },
+  {
+    brand: 'Huawei',
+    model: 'P30 Pro',
+  },
```
Comment (machour) [inline deviceinfo.js]: Could you please move these additional lines to the bottom (keeping the whole array alphabetically sorted)

Comment (Taym95) [reply deviceinfo.js]: Hi,  I added it after `Huawei`, it's already sorted alphabetically, right?

Comment (Taym95) [reply deviceinfo.js]: I copied the array to javascript console and tried `.sort()` function and the array still the same.

Comment (mikehardy) [reply deviceinfo.js]: I do like that it is alphabetized but I'd be hard-pressed to try harder than actually testing sort() output :-)

**Pull request merged**